Hackney.Shared.Finance.Tests/Dockerfile

```diff
@@ -0,0 +1,24 @@
+﻿FROM mcr.microsoft.com/dotnet/core/sdk:3.1
+
+# disable microsoft telematry
+ENV DOTNET_CLI_TELEMETRY_OPTOUT='true'
+
+ARG LBHPACKAGESTOKEN
+ENV LBHPACKAGESTOKEN=$LBHPACKAGESTOKEN
```

Comment:
I'm pretty sure the Line 7 is redundant.
As can be seen from this line: `LBHPACKAGESTOKEN=$LBHPACKAGESTOKEN`, the LBHPACKAGESTOKEN is already available as an environment variable as it's being used with the $ variable access syntax.

Reply:
removed line 7